<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Busca_cta_diario</class>
 <widget class="QDialog" name="Busca_cta_diario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buscar Cuenta en Diario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Concepto:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="concepto_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Documento:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="doc_lineEdit"/>
     </item>
     <item>
      <widget class="QCheckBox" name="fecha_checkBox">
       <property name="text">
        <string>Fecha</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="fecha_dateEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Debe:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="debe_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Haber:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="haber_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="diario_tableWidget">
     <column>
      <property name="text">
       <string>Cuenta</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Asiento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Concepto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Debe</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Haber</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Documento</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refrescar_pushButton">
       <property name="text">
        <string>&amp;Refrescar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/rayo.png</normaloff>:/iconos/graf/rayo.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_pushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptar_pushButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/check2.png</normaloff>:/iconos/graf/check2.png</iconset>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>concepto_lineEdit</tabstop>
  <tabstop>doc_lineEdit</tabstop>
  <tabstop>fecha_checkBox</tabstop>
  <tabstop>fecha_dateEdit</tabstop>
  <tabstop>debe_lineEdit</tabstop>
  <tabstop>haber_lineEdit</tabstop>
  <tabstop>diario_tableWidget</tabstop>
  <tabstop>cancel_pushButton</tabstop>
  <tabstop>aceptar_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancel_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Busca_cta_diario</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aceptar_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Busca_cta_diario</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>746</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
